src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -503,6 +503,18 @@ def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, na
 
 def k8s_get_credentials(name=None, resource_group_name=None, dns_prefix=None, location=None, user=None,
                         path=os.path.join(os.path.expanduser('~'), '.kube', 'config'), acs_info=None):
+    """Create a new Acs.
+    :param name: The name of the cluster.
+    :type name: str
+    :param resource_group_name: The name of the resource group. The name
+     is case insensitive.
```

Comment:
You can skip the `is case insensitive`, it is well known the group name should be handled case insensitively. Also, by default, argument values should be handled as case insensitive if we can like those will be part of url, enum choices, etc